tests/foreman/cli/test_registration.py

```diff
@@ -106,9 +106,8 @@ def test_host_registration_end_to_end(
     assert rhel_contenthost.subscription_config['server']['port'] == CLIENT_PORT
 
 
-def test_upgrade_katello_ca_consumer_rpm(
-    module_org, module_location, target_sat, rhel7_contenthost
-):
+@pytest.mark.rhel_ver_match('7')
```

Comment:
Can we update it to use default_rhel_version now? or is EL7 still a requirement?

Reply:
We are already testing this on all rhel versions in multiple tests but it is running on VMs so I wanted to keep the version as it is and it is also linked to a bug so did not want to change it.